src/loader/networks/sdex-chain.ts

```diff
@@ -28,7 +45,182 @@ export class SdexChain {
         quote: string;
         poolIdx: number;
       }[];
-    }>(gglPools, { limit });
+    }>(gqlPools, { limit });
+  }
+
+  public async queryUserPositions(user: string) {
+    return this.queryFromSubgraph<LiquidityChangesResponse>(gqlLiquidityChanges, { user });
+  }
+
+  private async executeMulticall(calls: Array<{ target: string; callData: string }>) {
+    const results = await this.multicall.tryAggregate.staticCall(true, calls);
+    if (results[0].success && results[0].returnData) {
+      return results[0].returnData;
+    } else {
+      throw new Error('Multicall failed or returned empty');
+    }
+  }
```

Comment:
Could be moved to `Chain` class and accessed by `this.context.executeMulticall` as shared helper later.